<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::calibrationwidget</class>
 <widget class="QWidget" name="snowgui::calibrationwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>179</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="resolutionField">
        <property name="text">
         <string>10000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="numberLabel">
        <property name="text">
         <string>Points:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="numberField">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="snowgui::CalibrationDisplayWidget" name="calibrationdisplayWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="calibrationIDLabel">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="qualityField">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="calibrationIDField">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="resolutionLabel">
        <property name="text">
         <string>Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="qualityLabel">
        <property name="text">
         <string>Quality:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="intervalLabel">
        <property name="text">
         <string>Interval:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="intervalField">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="calibrateButton">
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="databaseButton">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detailButton">
        <property name="text">
         <string>Detail</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::CalibrationDisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">CalibrationDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
